src/EditorFeatures/CSharpTest/InitializeParameter/AddParameterCheckTests.cs

```diff
@@ -30,6 +30,33 @@ public async Task TestEmptyFile()
 
         [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsInitializeParameter)]
         public async Task TestSimpleReferenceType()
+        {
+            await new VerifyCS.Test
+            {
+                LanguageVersion = LanguageVersion.Preview,
+                TestCode = @"
+using System;
+
+class C
+{
+    public C([||]string s)
```

Comment:
Please add a test where we're starting with `string s!!` (fix should not be offered)